### ISSUE TITLE
Why is the new version is not published?

### ISSUE THREAD

**Issue by ezhaka** [craigburke/karma-gradle#36]
The current version `1.4.4` doesn't work with the newest version of `gradle-node-plugin`. And it's already fixed in master, but is not published to Gradle plugin repository yet. Is there any problem to publish?

**Comment by vricosti** [comment]
Yes can we ask for pull request because I would like to use com.github.node-gradle:gradle-node-plugin:2.2.4 instead of com.moowork.gradle:gradle-node-plugin:1.0.0.
Will you publish a new version ?
Can I fork and publish a new version if nobody does  ?
